changelogs/fragments/7463.yml

```diff
@@ -0,0 +1,2 @@
+feat:
+- Add MDS support along with a few cleanup and tests update ([#7463](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7463))
```

Comment:
wrong PR number?

Reply:
this commit is cherry picking previous commit after revert, therefore it contains this as part of the pervious commit, the new changelog was added in the description and auto generated